deploy-keytab.yml

```diff
@@ -0,0 +1,67 @@
+#*  This playbook exports the kerberos host keytab from IPA and deploys
+#* it on a host to /etc/krb5.keytab.
+#*  It requires your Ansible user to have SSH access to the IPA server
+#* and a valid kerberos ticket with the required export permissions
+#* already issued.
+#*  No arguments are needed.
+
+- name: Export Kerberos keytab from IPA and deploy it on host
+  hosts: host.Virtual,&os.debian_buster
```

Comment:
I don't really like this. Should we make a new class `host.IPAClient` or something? Or any other suggestion how to restrict this to designated hosts?